### ISSUE TITLE
Different behaviour between insomnia-inso from npm and binary release

### ISSUE THREAD

**Issue by michbeck100** [getinsomnia/insomnia#4432]
### Expected Behavior
Generate the same declarative config.

### Actual Behavior
When running `inso generate` the two inso versions create different declarative configs.

### Reproduction Steps

Example OpenAPI spec:

```yaml
openapi: 3.0.0
info:
  version: 1.0.0
  title: Sample-API
servers:
  - url: https://example.com
paths:
  /list:
    get:
      description: Returns a list of stuff
      responses:
        '200':
          description: Successful response
```
 Run `inso generate config api-docs.yml -o kong-api-docs.yml` with both inso cli types and compare the result.

### Is there an existing issue for this?

- [x]  I have searched the issue tracker for this problem.

### Additional Information
When using hyphens in `info.title` (here "Sample-API") insomnia-inso from npm keeps the title as service name and the binary release converts every hyphen to underscore. 

### Insomnia cli version
2.4.1

### What operating system are you using?
Linux in Kubernetes

### Installation method
Downloaded using curl and installed using npm

### Last Known Working Insomnia version
The version from npm is working

**Comment by filfreire** [comment]
I was able to reproduce this.

1) installed via npm, `npm install -g insomnia-inso`

2) downloaded and unzipped [macos build](https://github.com/Kong/insomnia/releases/tag/lib%402.4.1)

3) Tried to covert above file with each:
```
issue4432 $inso generate config api-docs.yml -o via-npm.yml
Configuration generated to "via-npm.yml".                                                                                                                                                                    
issue4432 $./inso generate config api-docs.yml -o via-download.yml
Configuration generated to "via-download.yml". 
```

Result:

```shell
issue4432 $diff via-npm.yml via-download.yml 
3c3
<   - name: Sample-API
---
>   - name: Sample_API
13c13
<         name: Sample-API-list-get
---
>         name: Sample_API-list-get
issue4432 $
```

(Was also able to reproduce with latest [alpha build of inso](https://github.com/Kong/insomnia/releases/tag/lib%402.4.2-alpha.2))

cc @wdawson  - not yet sure why it is happening, but we know at least it is reproducible.

**Comment by filfreire** [comment]
Hey @michbeck100 sorry for the wait regarding this issue.

We have identified the problem and prepared a fix (see PR #4657).

The fix should be included in tomorrow's beta release.

**Comment by filfreire** [comment]
Hi @michbeck100, we've launched a new beta today, and from [Inso CLI version 2.11.0-beta.0](https://github.com/Kong/insomnia/releases/tag/lib%402.11.0-beta.0) onwards, the behaviour between npm package and release binary should be similar.

Thank you for reporting this, feel free to report any other issues in the future!
